conda-recipe/vcdat/meta.yaml

```diff
@@ -20,4 +20,4 @@ source:
   git_rev: v0.0.1
```

Comment:
Do you want me to tag a new release in vcdat master 0.0.2? before or after we merge this? Does @SandiSamuel need to change anything in the file before I merge? Conda Recipes are new to me.